src/Compiler/SyntaxTree/SyntaxTree.fs

```diff
@@ -1015,6 +1015,14 @@ type SynAttribute =
         AppliesToGetterAndSetter: bool
 
         Range: range
+
+        LessRange: range option
+
+        TypeArgs: SynType list
+
+        CommaRanges: range list
+
+        GreaterRange: range option
```

Comment:
This feels like a `Trivia` node that @nojaf might have opinions on?

Reply:
Yes, `SynType.App`, `SynType.LongIdentApp` and `SynExpr.TypeApp` use those 3 range fields too.

Reply:
`SynType.App, SynType.LongIdentApp and SynExpr.TypeApp` pre-date `SynTrivia`, I think for a new concept it make sense to have these as trivia.
It might also make sense to introduce a new type for `TypeName : SynLongIdent`.